solution/backend/content_search/serializers.py

```diff
@@ -24,16 +24,8 @@ class ContentListSerializer(DetailsSerializer, serializers.Serializer, ):
 
 class ContentSearchSerializer(ContentListSerializer, ):
     document_name_headline = HeadlineField()
-    summary_headline = serializers.SerializerMethodField()
-
-    def get_summary_headline(self, obj):
-        check_string = "search-highlight"
-        if check_string in obj.description_headline:
-            return obj.description_headline
-        elif obj.content and check_string in obj.content_headline:
-            return "..." + obj.content_headline + "..."
-        else:
-            return obj.summary_string
+    summary_headline = HeadlineField()
+    content_headline = HeadlineField()
```

Comment:
All of the Django/python changes here are rolling back the work done in EREGCSC-2340 since they no longer need `summary_headline` to conditionally be different things at different times.  That logic is now on the front end